File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F06.P2.kt
fun caller(): Int {
    // BEGIN PROBE F06.P2
    fun divmod2(a: Int, b: Int) = a / b to a % b
    val (q, r) = divmod2(7, 3)
    return q + r
    // END PROBE F06.P2
}

fun main() {
    println(caller())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F06.P2.cpp
#include <cstdio>
#include <utility>

std::pair<int, int> divmod2(int a, int b);

int caller() {
// BEGIN PROBE F06.P2
    auto [q, r] = divmod2(7, 3);
    return q + r;
}

std::pair<int, int> divmod2(int a, int b) { return {a / b, a % b}; }
// END PROBE F06.P2

int main() { std::printf("%d\n", caller()); }
